homework-submission/week3-homework/react-hyf/src/App.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import Spinner from './spinner';
+import './App.css';
+
+class App extends Component {
+  state = {
+    personInfos: [],
+    personInfo: {},
+    isLoading: true,
+  };
+  // https://uinames.com/api/?amount=10
+  componentWillMount() {
```

Comment:
`componentWillMount` is deprecated, use `componentDidMount` instead! https://reactjs.org/docs/react-component.html#unsafe_componentwillmount